src/p2p/behaviour.rs

```diff
@@ -84,15 +84,15 @@ impl<Types: IpfsTypes> NetworkBehaviourEventProcess<KademliaEvent> for Behaviour
                     }
                     GetClosestPeers(Ok(GetClosestPeersOk { key: _, peers })) => {
                         for peer in peers {
-                            info!("kad: peer {} is close", peer);
+                            debug!("kad: peer {} is close", peer);
```

Comment:
I remember thinking this earlier but not commenting; why is the key not printed out here? If it is because some other log message always prints it out, perhaps it could be commented here? Same question applies for other places which don't report the key, like the `GetClosestPeers(Err(...))` below.

Reply:
In this case we're just asking for peers closest to us, so the key is just our node's id - I just didn't think it's that interesting in that case.

Reply:
Aah yes ... Didn't remember the implicit "query is for our peerid". Perhaps it could commented here what the expectation is?